### PR TITLE
Update the default public key

### PR DESCRIPTION
**Pull request by hsmj1412**
OpenSSH 7.0 disables ssh-dss keys by default,
so set the default ssh-rsa

Ref: https://www.openssh.com/txt/release-7.0

Signed-off-by: Junxiang Li <junli@redhat.com>

### CODE REVIEW

**Comment by hsmj1412** [comment]
@sathnaga @balamuruhans @ldoktor @rdkdd @luckyh @hereischen Please help check and confirm this PR will not block your test

**Comment by luckyh** [comment]
The change looks good to me, thanks.

File: virttest/ssh_key.py [hunk 1]
```diff
@@ -32,19 +32,19 @@ def get_public_key():
...
-        print('DSA keypair found, using it')
-        public_key_path = dsa_public_key_path
-
-    elif has_rsa_keypair:
+    if has_rsa_keypair:
         print('RSA keypair found, using it')
         public_key_path = rsa_public_key_path
 
+    elif has_dsa_keypair:
+        print('DSA keypair found, using it')
```
Comment (balamuruhans) [inline virttest/ssh_key.py]: can you help to make use of logging instead here and elsewhere.

Comment (hsmj1412) [reply virttest/ssh_key.py]: Updated

File: virttest/ssh_key.py [hunk 2]
```diff
@@ -32,31 +32,31 @@ def get_public_key():
...
-    else:
-        print('Neither RSA nor DSA keypair found, creating DSA ssh key pair')
-        process.system('ssh-keygen -t dsa -q -N "" -f %s' %
-                       dsa_private_key_path)
+    elif has_dsa_keypair:
+        logging.info('DSA keypair found, using it')
         public_key_path = dsa_public_key_path
 
+    else:
+        logging.info('Neither RSA nor DSA keypair found, creating DSA ssh key pair')
```
Comment (xutian) [inline virttest/ssh_key.py]: should be "creating RSA  ssh key pair ", right?

Comment (hsmj1412) [reply virttest/ssh_key.py]: Fixed, thx

**Comment by balamuruhans** [comment]
Thanks all. let us merge it.

**Pull request merged**